ethereum/rpc/namespaces/eth/api.go

```diff
@@ -1084,3 +1078,35 @@ func (e *PublicAPI) setTxDefaults(args rpctypes.SendTxArgs) (rpctypes.SendTxArgs
 
 	return args, nil
 }
+
+// getAccountNonce return the account nonce
```

Comment:
```suggestion
// getAccountNonce returns the account nonce for the given account address. If the pending value is true, it will iterate over the mempool (pending) txs in order to compute and return the pending tx sequence.
```

Reply:
fixed